1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,9 @@
 function readingList(books) {
   // Write your code here...
+  books.forEach((book) => {
+    document.createElement("p");
+    document.createElement("img");
+  });
```

Comment:
This is a good start, you will want to assign these elements to variables so that you can add `innerText` and `src` to them and then add them to the dom